### DISCUSSION TITLE
(Static) Routing & NAT support

### DISCUSSION THREAD

**Discussion by amunabv** [arendst/Tasmota#21448] [Ideas]
I think it would be nice to have some control over routes within the Tasmota configuration to allow more advanced networking. Would this be viable?

**Comment by sfromis** [comment]
Well, routing is basically the job of the router.....

**Reply by amunabv** [reply]
I think in possibilities, not limitations.

**Comment by Jason2866** [comment]
And we are thinking in use cases ;-)  Describe a scenario where this is usefull/needed. Niche use cases have a low chance to be considered to be implemented.
